### PR TITLE
Fix initialization issues

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
**Problem Description**
Some variables were not being initialized properly, therefore this is a pass on initializing some of the uninitialized variables.

Also, @bkueng raised that there is an issue with fixed size vectorizable Eigen types mentioned in https://eigen.tuxfamily.org/dox/group__TopicStructHavingEigenMembers.html that was addressed.

### CODE REVIEW

File: include/mavlink_interface.h [hunk 1]
```diff
@@ -84,6 +84,7 @@ enum class SensorSource {
 
 namespace SensorData {
 struct Imu {
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```
Comment (bkueng) [inline include/mavlink_interface.h:87]: Doesn't hurt to add it here, but would not strictly be required.

File: include/sensor_plugin.h [hunk 2]
```diff
@@ -57,4 +57,6 @@ class SensorPlugin {
   JSBSim::FGFDMExec *_sim_ptr;
   double _last_sim_time;
```
Comment (bkueng) [inline include/sensor_plugin.h:58]: Better initialize the things you can do here (less code):
```suggestion
  double _last_sim_time{0.};
```

File: src/sensor_mag_plugin.cpp [hunk 3]
```diff
@@ -52,7 +52,8 @@ SensorMagPlugin::SensorMagPlugin(JSBSim::FGFDMExec* jsbsim)
     : SensorPlugin(jsbsim),
       _noise_density(kDefaultNoiseDensity),
       _random_walk(kDefaultRandomWalk),
-      _bias_correlation_time(kDefaultBiasCorrelationTime) {
+      _bias_correlation_time(kDefaultBiasCorrelationTime),
+      _bias(Eigen::Vector3d::Zero()) {
```
Comment (bkueng) [inline src/sensor_mag_plugin.cpp:56]: Same here: better initialize in the class.

**Comment by Jaeyoung-Lim** [comment]
@bkueng Thanks for the review! I have moved all the initializations to the headerfile

File: include/sensor_airspeed_plugin.h [hunk 4]
```diff
@@ -54,7 +54,6 @@ class SensorAirspeedPlugin : public SensorPlugin {
   double getAirspeed();
   double getAirTemperature();
 
-  std::default_random_engine _random_generator;
   std::normal_distribution<double> standard_normal_distribution_;
-  double diff_pressure_stddev_;
+  double diff_pressure_stddev_{0.01f};
```
Comment (bkueng) [inline include/sensor_airspeed_plugin.h:58]: Should be `double`:
```suggestion
  double diff_pressure_stddev_{0.01};
```

**Review by bkueng — APPROVED** [review]

File: include/sensor_airspeed_plugin.h (hunk 4 above)
Comment (Jaeyoung-Lim) [reply include/sensor_airspeed_plugin.h]: Thanks!

**Pull request merged**